### PR TITLE
feat: state channel details hints

### LINKED ISSUE #232: State channel details page hints
Display hints on state channels details page.

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves #232 

Implements state channel details hints without the transactions section (as agreed with @michele-franchi)

## Demo
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

File: src/utils/hints/stateChannelsHints.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 export const stateChannelsHints = {
   stateChannelId: 'Unique identifier of the State Channel.',
   status: 'When status is "Open" transactions can be executed inside the State Channel, outside the blockchain network.',
-  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx. Responder is the account that joined the State Channel.',
   onChainUpdates: 'Amount of on-chain transactions within the State Channel.',
   locked: 'Currently deposited amount to the State Channel by its participants.',
   lastUpdated: 'Block height and estimated date and time of the last interaction with the State Channel.',
   lastTxType: 'Last transaction\'s type of the State Channel.',
+  createTransaction: 'The transaction that initialized the State Channel.',
```
Comment (janmichek) [inline src/utils/hints/stateChannelsHints.js:9]:
```suggestion
  createTransaction: 'The transaction that initiated the State Channel.',
```
?

File: src/utils/hints/stateChannelsHints.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 export const stateChannelsHints = {
   stateChannelId: 'Unique identifier of the State Channel.',
   status: 'When status is "Open" transactions can be executed inside the State Channel, outside the blockchain network.',
-  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx. Responder is the account that joined the State Channel.',
   onChainUpdates: 'Amount of on-chain transactions within the State Channel.',
   locked: 'Currently deposited amount to the State Channel by its participants.',
   lastUpdated: 'Block height and estimated date and time of the last interaction with the State Channel.',
   lastTxType: 'Last transaction\'s type of the State Channel.',
```
Comment (janmichek) [inline src/utils/hints/stateChannelsHints.js:8]:
```suggestion
  lastTxType: 'Type of the last transaction in the State Channel.',
```
I know it was my part of the work, but now when I see it, I would change it like this

File: src/utils/hints/stateChannelsHints.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+  createTransaction: 'The transaction that initialized the State Channel.',
+  initialAmount: 'The amount of tokens that were deposited to the State Channel by the initiator account.',
+  initiator: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  responder: 'The participating account that joined the State Channel.',
+  lastKnownRound: 'Round is a value used to order different channel states. The greater the value, the more recent the state is.',
```
Comment (janmichek) [inline src/utils/hints/stateChannelsHints.js:13]:
```suggestion
  lastKnownRound: 'Round is a value to order different channel states. The greater the value, the more recent the state is.',
```

**Review by janmichek — APPROVED** [review]
Looks fresh and approving. Giving just some minor content suggestions 🍾

File: src/utils/hints/stateChannelsHints.js (hunk 3 above)
Comment (michele-franchi) [reply src/utils/hints/stateChannelsHints.js:13]: What about: `Round is a value used to order various updates in the State Channel, which should be incremented with each off-chain update.`

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Please have a look at my suggestions.

**Comment by lukeromanowicz** [comment]
Thanks @michele-franchi @janmichek for the great suggestions. All of them are included

**Review by michele-franchi — APPROVED** [review]
Good job!

**Pull request merged**